Content.Server/_CM14/Scoping/ScopeSystem.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Numerics;
+using Content.Server.Movement.Systems;
+using Content.Shared._CM14.Scoping;
+using Content.Shared.Camera;
+using Robust.Server.GameObjects;
+using Robust.Shared.Player;
+
+namespace Content.Server._CM14.Scoping;
+
+public sealed class ScopeSystem : SharedScopeSystem
+{
+    [Dependency] private readonly ViewSubscriberSystem _viewSubscriber = default!;
+    [Dependency] private readonly SharedTransformSystem _transformSystem = default!;
+    [Dependency] private readonly ContentEyeSystem _contentEye = default!;
+    [Dependency] private readonly EyeSystem _eye = default!;
+
+    protected override void StartScopingCamera(EntityUid user, ScopeComponent scopeComponent)
+    {
+        var xform = Transform(user);
+        if (!HasComp<CameraRecoilComponent>(user))
+            return;
+
+        const float smallestViewpointSize = 15;
+
+        var cardinalVector = xform.LocalRotation.GetCardinalDir().ToVec();
+        var targetOffset = cardinalVector * ((smallestViewpointSize * scopeComponent.Zoom - 1) / 2);
+
+        _eye.SetOffset(user, targetOffset); // apparently this is required for aim to work properly
+        _contentEye.SetZoom(user, Vector2.One * scopeComponent.Zoom, true);
+
+        var scopeToggleEvent = new CMScopeToggleEvent(GetNetEntity(user), targetOffset);
+        RaiseNetworkEvent(scopeToggleEvent, user);
+
+        if (TryComp(user, out ActorComponent? actorComp))
+        {
+            // add cardinal vector, until better pvs handling is introduced here
+            var loaderId = Spawn(scopeComponent.PvsLoaderProto, _transformSystem.GetMapCoordinates(xform).Offset(targetOffset + cardinalVector * 2));
+            scopeComponent.PvsLoader = loaderId;
+            _viewSubscriber.AddViewSubscriber(loaderId, actorComp.PlayerSession);
+        }
```

Comment:
I'm looking into this more closely now, I think you should be able to do what "Watch Xenomorph" does to move the player's view:
```csharp
_eye.SetTarget(watcher, watch, watcher);
_viewSubscriber.AddViewSubscriber(watch, player);
```

That should let you avoid having to change eye offsets and having to send a networked event.